Formula/bwa.rb

```diff
@@ -0,0 +1,21 @@
+class Bwa < Formula
+  desc "Burrow-Wheeler Aligner for pairwise alignment of DNA"
+  homepage "https://github.com/lh3/bwa"
+  url "https://github.com/lh3/bwa/releases/download/v0.7.16/bwa-0.7.16a.tar.bz2"
+  sha256 "8fecdb5f88871351bbe050c18d6078121456c36ad75c5c78f33a926560ffc170"
+  head "https://github.com/lh3/bwa.git"
+
+  def install
+    system "make", "CC=#{ENV.cc}", "CFLAGS=#{ENV.cflags}"
+    bin.install "bwa"
+    doc.install "README.md", "NEWS.md"
```

Comment:
This line should no longer be necessary. These documentation files are installed by default now by `brew`. Can you check that after removing this line, these two files are still installed in the keg?